python/lsst/meas/astrom/astrometryNetDataConfig.py

```diff
@@ -3,6 +3,9 @@
 mechanism, but we need nested lists, so we do this home-brew version
 instead.
 '''
+from past.builtins import execfile
```

Comment:
We have generally used `past.builtins` to give us compatibility with behavioral changes between 2 and 3. In particular using `long` and `basestring` as they are providing semantics to python 2 that do not exist on 3. `execfile` is not like that and it just a function that needs to be replaced with a modern approach.